src/server.py

```diff
@@ -422,6 +416,25 @@ def run_create_jobtypes_query():
         query = "insert into uniquejobs (platform, buildtype, testtype) values ('%s', '%s', '%s')" % (j[0], j[1], j[2])
         cursor.execute(query)
 
+    # Add in preseed.json
+    preseed = []
+    with open(os.path.join(SCRIPT_DIR, 'preseed.json'), 'r') as fHandle:
+        preseed = json.load(fHandle)
+
+    for job in preseed:
+        found = False
+        for j in jobtypes:
+            if j[0] == job['platform'] and \
+               j[1] == job['buildtype'] and \
+               j[2] == job['name']:
+                found = True
+                break
+        if found:
+            continue
+
+        query = "insert into uniquejobs (platform, buildtype, testtype) values ('%s', '%s', '%s')" % (job['platform'], job['buildtype'], job['name'])
+        cursor.execute(query)
+
```

Comment:
Checking for duplicates is fine. We insert in uniquejobs just before the additional code too: https://github.com/dminor/ouija/blob/ignorelist/src/server.py#L415
I would like to add all the jobs to jobtypes from preseed, and then insert in uniquejobs only once.